pyproject.toml

```diff
@@ -30,6 +30,7 @@ dependencies = [
     "openai==0.27.4",
     "pip==21.2.4",
     "pre-commit>=4.2.0",
+    "pyfakefs>=5.8.0",
```

Comment:
Make sure to differentiate between package dependencies that are used during development and those that are required when the code is installed as a package.

Dev dependency: `uv add --dev ...`

Package dependency: `uv add ...`